tests/compatibility/test_compatibility.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from great_expectations.compatibility.not_imported import NotImported
+
+# TODO: parametrize test_error_raised_when_optional_import_not_installed over all optional dependencies
+# TODO: Test to make sure we don't have optional dependencies that are not in our set of tests in test_error_raised_when_optional_import_not_installed
+
+
+@pytest.mark.compatibility
+def test_error_raised_when_optional_import_not_installed():
+    from great_expectations.compatibility.pyspark import pyspark
+
+    breakpoint()
```

Comment:
This looks like a stray breakpoint

Reply:
😅 Removed in https://github.com/great-expectations/great_expectations/pull/7898/commits/42383734c7093ec2e58f3fcb5eb5c04332b8e4c6